### PR TITLE
Improve texts when no PRs found

### LINKED ISSUE #7035: prow: user dashboard: when no PRs are open, message is not very pretty
no-prs

The rest of the dashboard is really pretty, this part is not :)

/kind bug
/area prow
/p0
/cc @BenTheElder 
/assign @qhuynh96

### PR DESCRIPTION
**Pull request by qhuynh96**
* Fixes #7035
Makes user dashboard great (again)!

9fs5cp4ozgj

### CODE REVIEW

**Comment by BenTheElder** [comment]
squash?

File: prow/cmd/deck/static/user-script.js [hunk 1]
```diff
@@ -669,3 +663,23 @@ function createIcon(iconString, tooltip = "", styles = [], isButton = false) {
 
     return container;
 }
+
+/**
+ * Create a simple message with an icon.
```
Comment (BenTheElder) [inline prow/cmd/deck/static/user-script.js]: we aren't setting an icon anymore?

Comment (qhuynh96) [reply prow/cmd/deck/static/user-script.js]: The login message needs an :smile: icon (or not).

**Pull request merged**